install_config/router/default_haproxy_router.adoc

```diff
@@ -1457,6 +1457,129 @@ access any pod in the cluster. If isolation is needed in this case, then do not
 add routes across the namespaces.
 ====
 
+[[using-the-dynamic-configuration-manager]]
+== Using the Dynamic Configuration Manager
+
+You can configure the HAProxy router to support the dynamic configuration
+manager.
+
+The dynamic configuration manager brings certain
+types of routes online without requiring any HAProxy reload downtime. It handles any route and endpoint life
+cycle events such as route and endpoint `addition|deletion|update`.
+
+Enable the dynamic configuration manager by setting the
+`ROUTER_HAPROXY_CONFIG_MANAGER` environment variable to `true`:
+
+----
+$ oc set env dc/<router_name> ROUTER_HAPROXY_CONFIG_MANAGER='true'
+----
+
+If the dynamic configuration manager is unable to dynamically configure HAProxy,
+it forces a configuration re-write and a HAProxy process reload. For example, if
+a newly added route contains custom annotations that would cause a drift from
+the standard configuration (custom timeouts) or if the route requires custom TLS
+configuration.
+
+The dynamic configuration internally uses the haproxy socket and configuration
+API with a pool of pre-allocated routes and back end servers. The pre-allocated
+pool of routes is created using route blueprints. The default set of blueprints
+supports unsecured routes, edge secured routes (without any custom TLS
+configuration), and passthrough routes.
+
+[IMPORTANT]
+====
+`re-encypt` routes require custom TLS configuration information and so are not
```

Comment:
s/so/

Reply:
Wait, do we really mean "and so are not supported out of the box?" Do you mean something like

```
`re-encypt` routes require custom TLS configuration information, so you cannot use them directly.

You can extend the blueprints that the dynamic configuration manager uses 
by setting the `ROUTER_BLUEPRINT_ROUTE_NAMESPACE` and optionally the
`ROUTER_BLUEPRINT_ROUTE_LABELS` environment variable values. All routes, or the
routes that match the route labels, in the blueprint route namespace are
processed as custom blueprints similar to the default set of blueprints.

To use `re-encypt` routes or routes
that use custom annotations or routes with custom TLS configuration, define them
through blueprints.
```